src/components/binding/binding-renderer.js

```diff
@@ -186,7 +192,8 @@ genotet.BindingRenderer.prototype.init = function() {
  * @private
  */
 genotet.BindingRenderer.prototype.getBindingRanges_ = function() {
-  if (this.data.overviewRangeChanged) {
+  if (this.data.overviewRangeChanged || !this.scaleSet_) {
```

Comment:
Your usage of this.scaleSet_ seems to be only ensuring that the code below is run at least once, because you never set this.scaleSet_ back to false again. Therefore this variable is very suspicious and seems hacky to me.

this.data.overviewRangeChanged would have been set to true on the first data load. From previous discussion you mentioned that when multiple track are loaded the second loaded track would have it over-written to false. Why don't you just prevent that overwriting and then set overviewRangeChanged back to false here (so that you use overviewRangeChanged as a flag to control when to update scale)?